src/metabase/api/x_ray.clj

```diff
@@ -56,6 +58,26 @@
                                                  max_computation_cost)})
        fe/x-ray))
 
+(api/defendpoint GET "/async/table/:id"
```

Comment:
this should probably be called `/result/table/:id` or something similar

Reply:
That's not the result but submitting a computation. You get result via GET /async/:id (I don't think there is much point in having multiple result fetching endpoints)